apps/website/src/lib/tiptap/extensions/command.ts

```
@@ -49,6 +52,75 @@ export const Command = Extension.create({
+                  }
+                }
+
+                if (checkBefore) {
+                  const nodeBefore = $pos.nodeBefore;
+                  if (nodeBefore && nodeBefore.type === node.type) {
+                    tr.setSelection(NodeSelection.create(tr.doc, pos - nodeBefore.nodeSize));
+                    break;
+                  }
+                }
+              } catch {
+                continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)